packages/pages/src/common/src/function/types.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * Defines the functions and fields that are available to the serverless function.
+ * @public
+ */
+export interface FunctionModule {
+  /** The exported config function */
+  config?: FunctionConfig;
```

Comment:
Another similar comment. We need to check if these are relevant anymore for a user to define. I think the answer is going to be no.